pages/blog/swr-v2.en-US.mdx

```diff
@@ -0,0 +1,262 @@
+---
+image:
+description: 'We are excited to announce SWR v2.0 release, with various improvements of DX and new features like preloading, stale, optimistic update, etc.'
+date:
+---
+
+import Callout from 'nextra-theme-docs/callout'
+import Bleed from 'nextra-theme-docs/bleed'
+
+import Authors, { Author } from 'components/authors'
+import Video from 'components/video'
+
+# Announcing SWR 2.0
+
+<Authors date="">
+</Authors>
+
+We are excited to introduce SWR v2.0, including features to improve the use experience of your application.
+
+## What’s New
+
+### Better React 18 Support
+
+SWR now uses `useSyncExternalStore` and `startTransition` internally to improve the performance and manage states while integrating with React 18, which ensures stronger UI consistency under concurrent rendering mode. This change doesn't require any implicity updates on the interface perspective. Developers will benefit from the updates directly.
+
+### New Loading State Indicator
+
+`isLoading` is a new return value that indicates if there's an ongoing request and no "loaded data". Previously, you have to check `data` and `error` are `undefined`, but now you can use the `isLoading` value.
+
+```jsx
+import useSWR from 'swr'
+
+function Profile() {
+  const { data, isLoading } = useSWR('/api/user', fetcher)
+
+  if (isLoading) return <div>loading...</div>
+  return <div>hello {data.name}!</div>
+}
+```
+
+<Callout emoji="📝">
+  We have added the new [Understanding SWR](/docs/advanced/understanding) page to describe how SWR returns values, which includes the difference between `isValidating` and `isLoading`, and how to combine them to improve user exeprience.
+</Callout>
+
+
+### Lazy Updating
+
+`keepPreviousData` is a new option for keeping the previous fetched data. This improves the UX a lot when you fetch data based on continuous user actions, e.g. real-time search when typing. More information can be found [here](/docs/advanced/understanding#return-previous-data-for-better-ux).
+
+```jsx
+function Search() {
+  const [search, setSearch] = React.useState('');
+
+  const { data, isLoading } = useSWR(`/search?q=${search}`, fetcher, {
+    keepPreviousData: true
+  });
+
+  return (
+    <div>
+      <input
+        type="text"
+        value={search}
+        onChange={(e) => setSearch(e.target.value)}
+        placeholder="Search..."
+      />
+
+      <div className={isLoading ? "loading" : ""}>
+        {data?.products.map(item => <Product key={item.id} name={item.name} />)
+      </div>
+    </div>
+  );
+}
+```
+
```

Comment:
it's bit hard to find out it's a video on the website, since the cover picture is almost same color `#fff` with light mode background. adding a title to make it more explicit
```suggestion
#### Video Demo 
```